src/graphql/admin/schema.graphql

```diff
@@ -230,7 +226,6 @@ scalar Memo
 type Mutation {
   accountUpdateLevel(input: AccountUpdateLevelInput!): AccountDetailPayload!
   accountUpdateStatus(input: AccountUpdateStatusInput!): AccountDetailPayload!
-  accountsAddUsdWallet(input: AccountsAddUsdWalletInput!): WalletDetailsPayload @deprecated(reason: "All accounts have USD wallets by default")
```

Comment:
this requires an update of admin panel, also a couple of weeks ago we had a case with an account without stablesats wallet so maybe is better to check in db prev to this removal

Reply:
Iirc it was no longer used in the admin panel already

Reply:
confirmed that adminPanel is not using this method anymore